### ISSUE TITLE
Strapi 5.1.0 - couldn't install plugin

### ISSUE THREAD

**Issue by ViktorKim** [localazy/strapi-plugin#66]
Getting an error trying to install localizy plugin

<img width="634" alt="image" src="https://github.com/user-attachments/assets/1ee7e99c-e81b-4f48-adf3-ad1a2dd89612">

NODE: v20.17.0

<img width="375" alt="image" src="https://github.com/user-attachments/assets/f952179a-f90f-40e3-a6fa-a4166e587931">

**Comment by david-vaclavek** [comment]
The plugin does not currently support Strapi v5.

Implementing the Strapi v5 support is planned on Q1/2025.

Thanks for understanding, I'll keep you updated.

**Comment by maruthasalamr** [comment]
@david-vaclavek will this plugin is supported for strapi v5.5.1

**Comment by david-vaclavek** [comment]
@maruthasalamr Yes, as mentioned in my earlier comment, Strapi v5 support will be available by the end of Q1. It's one of our priorities, and we are currently working on it.

Thanks for your patience, I'll keep you posted.

**Comment by yunusyuksel** [comment]
I saw some comments related Strapi v5 migration. Is it available now ? I would love to get it as we want to migrate now our project to v5 :) 

Thanks

**Comment by ViktorKim** [comment]
I see that the new version was released, but we still have an issue

<img width="606" alt="Image" src="https://github.com/user-attachments/assets/6ffedbc9-82ea-4526-9fcd-5f76152ecc3c" />

**Comment by david-vaclavek** [comment]
@ViktorKim The beta is available now.

https://www.npmjs.com/package/@localazy/strapi-plugin/v/1.0.0-beta.1

We're waiting for your feedback.

**Comment by ViktorKim** [comment]
@david-vaclavek 
v1.0.0-beta.1 works, thanks
